Dockerfile

```diff
@@ -73,16 +74,18 @@ RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz
     rm -rf /chruby-* && rm -rf /usr/local/src/* && yum clean all
 
 ## GIT clone manageiq-appliance and self-service UI repo (SSUI)
-RUN git clone --depth 1 https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
-git clone --depth 1 https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
-ln -vs ${APP_ROOT} /opt/manageiq/manageiq
+RUN mkdir -p ${APP_ROOT} && \
+    mkdir -p ${APPLIANCE_ROOT} && \
+    mkdir -p ${SSUI_ROOT} && \
+    ln -vs ${APP_ROOT} /opt/manageiq/manageiq && \
+    curl -L https://github.com/ManageIQ/manageiq-appliance/tarball/${REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1 && \
+    curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
-## Create approot, ADD miq
-RUN mkdir -p ${APP_ROOT}
-ADD . ${APP_ROOT}
+## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
+# ADD . ${APP_ROOT}
```

Comment:
@bdunne good call on tarballs vs git clone, I think I got too used to ADD w/ .dockerignore for my image builds :smile: 

In my experience with ADD <tarball-url> this week, I actually could never get it to untar/expand automatically like it does with local tarballs, later I read on Dockerfile official docs that they do not untar/uncompress on remote sources.  

Those use cases should be the majority, at times it comes handy to build your own branch for quick demos, which also falls on dev case.